src/components/Staff/UpdateStaffForTimeRangeForm.tsx

```diff
@@ -83,127 +84,169 @@ export const UpdateStaffForTimeRangeForm = ({
       actualStaff: staffNumber
     };
     handleSubmit(ess);
+  };
+
+  function timesBy15Minutes(startHour: number): string[] {
+    const times: string[] = [];
+    for (let hour = startHour; hour < 24; hour++) {
+      for (let minute = 0; minute <= 45; minute += interval) {
+        times.push(`${hour.toString().padStart(2, '0')}:${minute.toString().padStart(2, '0')}`);
+      }
+    }
+    return times;
   }
-  const duration = moment.duration(endTime.diff(startTime));
-  const hours = Math.floor(duration.asHours());
-  const minutes = duration.minutes();
 
-  return <Box data-testid={`shift-staff-form`}
-              sx={{
-                paddingTop: '10px',
-                paddingLeft: '20px',
-                paddingBottom: '10px',
-                paddingRight: '20px',
-                width: '400px'
-              }}>
-    <Box sx={{display: 'flex', justifyContent: 'space-between'}}>
-      <Typography variant="h2" component="h2" fontWeight={"bold"}>Edit staff</Typography>
-      <IconButton
-        aria-label="close"
-        color="inherit"
-        size="small"
-        onClick={cancelHandler}>
-        <CloseIcon fontSize="inherit"/>
-      </IconButton>
-    </Box>
-    <Box sx={{paddingTop: '10px'}}>
-      <Typography variant="h3" component="h3">Date</Typography>
-      <Box sx={{paddingTop: '10px'}}>
-        <LocalizationProvider dateAdapter={AdapterMoment} adapterLocale={'en-gb'}>
-          <DatePicker sx={{backgroundColor: '#FFFFFF', width: '100%'}} label="Start Date" value={startDate}
-                      onChange={handleStartDateChange}
-                      format="DD MMMM YYYY"
-                      slots={{ textField: (params) => <TextField {...params} /> }}
-          />
-        </LocalizationProvider>
+  const timeOptions = timesBy15Minutes(0)
+
+  function timesBy15MinutesWithEnd(startHour: number): string[] {
```

Comment:
We decided to always show all the end times. It's a strange UX when the selected end time disappears because the new start time is later than it